core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/MockGoogleServicesDAO.scala

```diff
@@ -35,8 +36,8 @@ class MockGoogleServicesDAO(groupsPrefix: String,
   private val groups: TrieMap[RawlsGroupRef, Set[Either[RawlsUser, RawlsGroup]]] = TrieMap()
   val policies: TrieMap[GoogleProjectId, Map[String, Set[String]]] = TrieMap()
 
-  val accessibleBillingAccountName = RawlsBillingAccountName("billingAccounts/firecloudHasThisOne")
-  val inaccessibleBillingAccountName = RawlsBillingAccountName("billingAccounts/firecloudDoesntHaveThisOne")
+  val accessibleBillingAccountName = RawlsBillingAccountName("billingAccounts/123456-abcdef-789012")
```

Comment:
I had to update this because we now enforce the format of the billing account name (`billingAccounts/XXXXXX-XXXXXX-XXXXXX`)